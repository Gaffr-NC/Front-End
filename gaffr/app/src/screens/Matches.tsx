import React, { Component } from 'react';
import { View, Text } from 'react-native';

export default class Matches extends Component {
  static navigationOptions = {
    title: 'Matches'
  };

  render() {
    return (
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
        <Text>Matches!</Text>
      </View>
    );
  }
}
