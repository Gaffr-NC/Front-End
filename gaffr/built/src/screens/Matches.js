"use strict";
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
    result["default"] = mod;
    return result;
};
Object.defineProperty(exports, "__esModule", { value: true });
const react_1 = __importStar(require("react"));
const react_native_1 = require("react-native");
class Matches extends react_1.Component {
    render() {
        return (react_1.default.createElement(react_native_1.View, { style: { flex: 1, alignItems: 'center', justifyContent: 'center' } },
            react_1.default.createElement(react_native_1.Text, null, "Matches!")));
    }
}
Matches.navigationOptions = {
    title: 'Matches'
};
exports.default = Matches;
