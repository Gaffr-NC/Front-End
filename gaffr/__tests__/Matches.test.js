import React from "react";
import renderer from "react-test-renderer";
import Matches from "../screens/Matches";

it("Matches renders without crashing", () => {
  const rendered = renderer.create(<Matches />).toJSON();
  expect(rendered).toBeTruthy();
});
it("Tests Matches against snapshot", () => {
  const tree = renderer.create(<Matches />).toJSON();
  expect(tree).toMatchSnapshot();
});
