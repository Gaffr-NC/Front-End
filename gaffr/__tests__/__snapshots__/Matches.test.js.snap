// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Matches against snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Matches!
  </Text>
</View>
`;
